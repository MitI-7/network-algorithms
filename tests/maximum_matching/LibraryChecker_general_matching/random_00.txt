181 1323 90
170 17
38 45
5 99
59 168
170 113
108 22
32 130
71 47
95 63
19 124
2 178
92 41
144 137
144 27
172 155
0 179
104 49
104 107
163 177
74 151
153 55
102 143
54 89
159 111
42 2
97 96
106 76
131 29
96 145
11 43
47 18
109 2
67 152
97 14
164 95
0 32
53 169
143 58
144 125
34 11
90 49
68 54
168 55
42 45
113 80
16 175
143 3
147 106
112 45
141 62
104 129
113 138
58 68
9 101
161 84
46 42
159 9
156 73
117 169
39 17
97 132
165 103
86 164
32 56
5 117
8 127
38 118
15 67
83 7
172 113
173 43
176 172
24 20
77 103
30 133
138 171
157 128
53 179
80 38
125 24
162 43
150 130
146 122
57 23
169 60
57 161
167 26
172 118
10 98
107 37
137 116
3 60
11 133
36 108
22 140
101 78
163 107
76 56
4 33
147 69
1 118
79 114
48 114
30 107
0 176
23 157
111 14
38 94
44 84
117 38
75 152
119 146
41 157
60 82
177 26
134 127
126 22
107 154
128 45
144 22
118 151
141 39
171 96
105 76
62 159
88 133
25 116
180 154
138 146
166 135
147 38
91 131
162 122
161 80
144 142
49 9
168 17
170 60
13 78
71 169
176 25
42 177
104 1
179 20
172 165
134 25
167 143
16 84
21 180
98 44
172 5
178 14
119 157
3 66
116 102
150 47
151 54
29 91
80 139
20 129
26 130
167 50
108 16
54 170
36 159
157 26
55 81
151 148
23 46
39 129
178 78
90 120
113 86
99 86
67 166
69 35
95 60
78 151
63 22
2 143
123 106
65 93
49 12
95 46
132 43
104 165
136 155
47 135
16 156
148 149
17 95
30 13
103 63
159 96
161 123
69 111
165 178
0 26
172 50
22 101
12 169
68 137
169 84
82 141
145 180
154 59
101 175
49 33
18 170
135 91
37 24
142 131
63 66
80 95
148 154
11 2
171 109
130 29
67 93
175 13
78 180
160 131
101 178
116 42
2 140
177 160
96 73
75 25
14 167
95 69
29 112
64 7
53 4
154 153
105 95
177 57
119 68
88 77
168 3
26 47
46 132
40 32
58 145
30 82
34 99
123 16
180 16
146 2
139 26
30 73
123 144
66 40
64 33
157 46
169 57
95 1
163 52
46 101
126 143
136 56
114 4
176 173
20 62
95 160
19 148
87 106
147 39
88 98
59 137
98 166
136 26
137 76
32 41
21 69
155 38
36 93
112 28
93 11
1 27
118 132
99 43
98 1
167 128
19 169
71 99
148 45
175 11
0 163
89 71
146 136
63 115
32 75
136 49
166 81
27 14
73 11
164 166
68 133
8 112
8 11
24 151
10 54
137 51
169 156
153 164
43 108
173 64
75 139
177 50
67 89
20 64
147 104
151 100
166 118
26 105
82 12
155 33
65 17
93 123
3 162
19 1
21 72
91 78
63 116
158 165
60 109
69 17
152 102
164 81
83 64
85 171
6 36
112 27
109 10
119 164
121 89
51 161
93 22
65 139
72 175
69 134
176 14
174 49
105 133
151 29
35 11
172 80
145 28
26 145
121 97
131 56
56 113
33 122
12 142
149 37
20 132
2 171
106 24
105 13
118 11
26 93
43 40
70 11
41 148
18 122
145 2
132 146
3 99
136 107
23 92
34 124
173 62
36 34
65 15
136 16
29 6
177 141
60 66
134 125
90 36
167 173
157 76
75 149
170 6
142 24
152 108
13 22
168 164
37 109
108 74
97 90
92 95
141 37
16 96
29 2
174 76
124 79
32 139
95 66
47 160
18 101
126 0
19 75
131 134
152 177
99 16
109 3
76 135
140 113
148 84
77 3
134 18
178 20
25 6
172 55
99 120
94 44
179 18
121 93
97 23
81 5
26 19
137 55
168 52
58 28
69 71
74 150
140 63
31 166
21 171
31 14
174 32
74 143
141 158
150 98
50 42
129 146
140 177
58 155
3 48
91 172
51 179
22 17
175 155
42 87
10 59
107 72
84 99
8 50
34 47
112 44
158 170
121 7
111 68
55 138
70 48
67 74
105 126
135 24
91 74
1 78
82 68
27 20
172 36
20 150
112 75
29 23
65 153
148 70
34 105
84 15
73 153
59 87
10 62
82 50
125 131
114 42
90 6
99 154
26 151
117 133
156 91
95 47
81 73
77 29
48 117
49 171
2 149
85 133
34 157
96 49
144 121
123 133
10 37
147 166
172 159
20 58
138 41
86 26
166 92
169 78
17 129
127 114
24 74
60 131
27 72
124 66
26 103
60 159
60 176
146 58
176 101
15 116
129 57
42 21
179 75
139 133
84 37
7 96
158 32
142 35
18 65
105 69
8 52
47 89
44 58
4 5
172 62
156 133
116 108
62 95
173 168
63 10
103 110
2 66
123 56
74 6
12 46
108 176
158 56
17 52
126 97
62 158
165 94
151 145
61 133
45 19
50 110
78 163
133 131
57 35
60 152
77 21
71 109
93 83
33 126
21 134
2 124
29 8
112 116
26 111
21 104
174 58
173 170
130 92
108 90
146 32
62 119
84 90
42 36
4 84
47 83
97 154
143 80
146 107
48 66
85 135
58 120
39 6
159 46
86 21
144 146
155 96
47 124
135 102
132 88
62 11
78 25
40 14
21 7
110 157
54 155
3 19
68 173
115 35
45 0
160 153
131 12
143 55
83 81
166 162
16 158
179 42
161 13
138 150
112 100
70 152
108 120
68 33
53 103
32 3
118 147
68 88
125 113
134 67
86 132
49 51
167 19
91 62
94 13
163 91
6 180
153 177
81 60
145 177
61 147
110 55
175 139
128 5
76 178
62 64
180 36
28 59
138 158
124 141
131 100
66 43
146 85
24 116
174 103
124 156
26 48
175 165
123 31
134 97
124 28
8 121
121 80
47 159
115 52
127 176
119 99
155 169
104 94
58 61
49 179
68 151
71 16
103 73
144 178
3 173
180 75
84 47
152 111
136 30
73 0
7 137
46 26
99 118
55 45
108 113
70 40
46 98
170 91
176 75
93 156
67 98
81 116
87 24
161 26
101 114
13 141
132 154
23 162
95 111
81 11
140 148
56 80
100 61
36 165
159 160
123 96
10 23
86 163
24 147
23 121
176 47
38 114
95 136
148 159
55 60
5 123
95 25
41 124
132 136
13 65
161 46
161 110
135 18
9 74
3 118
15 141
99 65
34 165
1 137
61 121
159 76
44 128
23 160
35 169
121 162
1 119
120 96
158 72
147 9
85 109
34 39
179 158
32 123
65 180
73 99
167 52
108 149
107 34
136 48
114 88
2 12
104 144
82 62
176 81
158 94
101 55
22 163
75 102
66 26
173 157
22 112
15 152
15 91
179 121
177 143
77 60
94 92
136 129
119 109
136 167
147 77
23 0
125 179
19 4
93 124
45 155
135 7
27 68
179 89
86 106
55 31
0 177
162 161
130 59
112 115
4 162
78 167
111 89
47 168
56 70
54 163
15 69
57 105
46 140
13 63
171 92
152 160
144 120
76 49
134 79
75 30
78 149
100 120
34 89
141 146
78 86
137 0
132 167
146 1
150 161
100 18
31 150
179 64
116 65
147 125
147 142
113 107
145 150
139 57
169 81
142 138
78 137
120 167
140 178
152 169
46 55
48 85
36 55
73 82
62 23
167 37
143 170
47 178
106 5
54 102
66 115
146 52
72 143
102 120
21 16
180 152
83 11
121 42
47 138
139 4
173 151
153 143
178 13
159 143
2 40
143 79
95 10
53 110
42 96
34 119
64 14
42 131
14 32
171 76
80 70
37 178
138 124
2 111
141 143
53 135
48 84
23 17
91 77
73 125
87 1
100 164
75 21
74 95
19 9
100 11
53 58
51 102
173 178
176 70
142 111
151 126
155 93
123 39
115 25
70 0
62 61
33 105
22 29
176 180
25 13
162 77
43 109
143 22
80 99
108 89
3 7
62 33
61 151
163 150
166 8
115 41
81 109
75 118
23 161
116 30
175 70
51 103
171 140
147 90
118 114
151 178
31 134
118 48
60 177
146 135
67 4
101 173
21 99
72 154
59 148
65 97
62 48
90 129
93 25
29 133
36 66
157 159
150 126
149 123
17 25
132 66
167 110
148 73
173 81
115 179
100 89
63 35
133 106
91 47
136 139
28 98
25 30
101 156
168 12
44 101
133 110
0 30
115 33
30 97
166 165
174 75
130 98
114 160
63 177
24 15
106 23
52 5
83 26
50 171
70 22
145 171
161 160
157 134
169 56
80 35
9 104
46 149
162 124
133 150
31 118
68 48
18 44
51 164
65 120
141 77
50 21
9 112
152 89
149 73
178 0
168 154
51 38
65 35
5 104
177 109
133 14
72 152
64 53
15 99
127 115
24 152
54 112
13 39
46 177
96 31
122 176
7 159
28 172
170 144
6 168
143 29
5 77
102 79
171 46
94 126
114 147
82 11
18 138
95 130
9 113
35 68
116 20
119 171
131 111
128 87
83 86
123 19
60 124
87 55
49 64
67 164
81 77
166 70
169 25
22 27
78 176
43 104
164 62
19 31
54 32
165 133
59 99
85 115
55 73
151 168
180 43
11 25
139 86
45 76
12 117
19 135
9 46
33 78
129 102
125 165
18 156
57 75
110 3
138 94
28 50
28 162
45 5
158 82
155 68
38 66
77 63
91 108
55 89
44 54
156 152
157 11
3 81
9 50
172 129
34 61
131 117
157 135
91 127
76 175
4 101
94 134
175 136
172 19
57 96
141 73
112 94
6 175
149 43
20 45
81 21
155 52
109 178
20 83
15 166
46 59
36 163
150 121
74 82
52 160
112 55
42 51
28 35
147 25
67 42
29 154
20 95
96 59
123 80
82 64
12 98
172 110
140 153
13 155
54 92
83 155
18 95
37 0
57 3
113 142
54 24
16 60
47 125
107 43
116 55
149 95
79 57
116 92
67 143
7 109
145 152
7 92
146 82
89 45
172 174
11 80
165 134
105 23
31 39
10 104
154 6
81 96
84 85
38 55
160 143
160 87
56 37
122 67
116 165
56 13
158 110
175 140
145 85
128 66
37 97
90 139
125 18
95 104
56 163
142 104
121 19
124 170
6 38
63 26
156 25
61 22
115 121
105 149
87 8
120 141
66 168
65 146
26 34
74 174
143 76
160 46
147 178
164 44
51 163
51 97
170 40
104 167
42 6
25 86
163 87
93 32
109 154
104 125
177 86
19 98
19 71
26 40
13 27
16 112
152 39
43 34
41 53
58 57
171 153
37 62
61 70
93 161
130 73
13 162
75 83
1 110
176 33
178 113
114 166
143 84
172 56
83 122
15 12
109 129
54 110
150 89
31 18
63 59
30 91
86 149
47 0
47 23
168 156
93 104
166 117
34 97
73 128
155 109
164 167
40 107
120 55
132 16
97 124
34 93
79 81
128 72
80 89
38 82
131 138
39 167
53 141
156 94
168 30
136 133
139 132
97 52
123 14
109 8
135 97
132 175
42 120
167 168
134 113
172 59
149 100
29 103
153 122
42 169
17 27
180 104
58 153
142 6
83 109
42 107
96 141
177 147
136 29
146 43
94 139
129 27
16 162
17 33
68 81
51 178
15 130
99 107
113 75
39 145
157 92
86 166
43 140
11 79
4 10
169 92
85 28
32 12
71 141
45 111
129 42
74 83
0 63
178 54
37 140
12 53
90 160
105 145
11 53
172 67
126 145
117 76
139 173
114 174
71 175
124 15
162 70
82 113
31 154
126 10
21 149
2 159
87 137
155 122
83 149
145 5
88 23
159 103
92 144
78 88
22 145
15 103
87 85
39 50
162 95
82 131
171 14
0 10
88 106
43 73
34 71
177 126
142 66
52 180
27 156
138 81
4 24
135 131
92 28
41 15
180 117
48 180
28 178
35 29
108 20
152 32
22 117
72 16
89 158
38 63