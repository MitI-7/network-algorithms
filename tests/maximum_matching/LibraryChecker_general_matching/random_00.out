90
0 179
1 118
2 178
3 143
4 33
5 117
6 36
7 83
8 127
9 101
10 98
11 43
12 49
13 78
14 97
15 67
16 175
17 170
18 47
19 124
20 150
21 180
22 140
23 57
24 125
25 116
26 167
27 144
28 112
29 131
30 136
31 166
32 130
34 105
35 69
37 107
38 155
39 141
40 66
41 92
42 46
44 84
45 128
48 114
50 172
51 137
52 163
53 169
54 89
55 153
56 76
58 68
59 168
60 82
61 147
62 119
63 95
64 173
65 93
70 148
71 109
72 158
73 156
74 151
75 174
77 103
79 134
80 139
81 164
85 171
86 132
87 106
88 133
90 120
91 135
94 165
96 145
99 154
100 149
102 129
104 142
108 176
110 157
111 159
113 138
115 121
122 146
123 161
126 177
152 160
